Packages/Sites/Neos.NeosIo/Resources/Private/Translations/en/Main.xlf

```diff
@@ -17,6 +17,9 @@
             <trans-unit id="fundingBadges.filterType.onetime" xml:space="preserve">
                 <source>One Time</source>
             </trans-unit>
+            <trans-unit id="fundingBadges.filterType.abo" xml:space="preserve">
```

Comment:
Yeah. There will be some further changes needed for the API to clean that up. 
But I need some sandstormers for that. I'll create a new issue.